src/styles/scss/shared.scss

```diff
@@ -22,3 +22,54 @@
   height: 1px;
   overflow: hidden;
 }
+
+.hide-768px {
+  @include breakpoint-s {
+    display: none;
+  }
+}
```

Comment:
As far as I can tell this class is only used once. Should this styling be included with the rest of the class there instead instead of adding a new functional class?

Reply:
Yup. I have removed the class from ```shared.scss``` and added a new class in ```list-materials.scss``` instead, where it should be used.